captcha-validation-service/src/main/java/io/mosip/esignet/captcha/constants/CaptchaErrorCode.java

```diff
@@ -0,0 +1,24 @@
+package io.mosip.esignet.captcha.constants;
+
+public enum CaptchaErrorCode {
+
+	INVALID_CAPTCHA_CODE("PRG-PAM-005", "Invalid Captcha entered"),
```

Comment:
lets change the errorcode to KER-CVS-001 and KER-CVS-002

Reply:
done anusha